modules/appeals_api/config/schemas/10182.json

```diff
@@ -75,8 +75,8 @@
       "type": "object",
       "additionalProperties": false,
       "properties": {
-        "issue":                     { "$ref": "#/definitions/nodCreateNonBlankString" },
-        "decisionDate":              { "$ref": "#/definitions/nodCreateDate" },
+        "issue":                     { "$ref": "#/definitions/nodCreateNonBlankString", "maxLength": 368 },
+        "decisionDate":              { "$ref": "#/definitions/nodCreateDate", "maxLength": 10 },
```

Comment:
I feel like the `maxLength` should be moved down to the `nodCreateDate` definition, since its pattern is looking for an exact number of characters anyway.

Since the pattern expects an exact number of characters, let's enforce `minLength` too.

Reply:
I'm not sure what you mean by "moved down to the nodCreateDate definition" - isn't this a specific length requirement on the `issues` key?